src/ui/components/shared/WorkspaceSettingsModal/WorkspaceMember.tsx

```diff
@@ -21,12 +21,19 @@ type WorkspaceMemberProps = {
 
 const memberRoleLabels: Record<WorkspaceUserRole, string> = {
   admin: "Admin",
+  // Not presented in the UI yet
+  contributor: "Contributor",
```

Comment:
What is this role going to be used for and how will it impact the UI?

I don't see an explanation of this role in SCS-1979 and semantically, I don't know what the difference between a "debugger" and a "contributor" would be.

Reply:
Ah, thanks for the link.

I think part of what bothers me here is that...these roles seem like they should either be hierarchical (e.g. viewer has read-only permissions, debugger has all of viewer's permissions _plus_ the ability to upload, admin has all of those permissions _plus_ the ability to manage the workspace) _or_ they should be...behaviors that you can mix and match.

Our UI seems to be kind of treating them like both (select implies hierarchy and checkbox implies behavior attributes). Why is it like that?

Reply:
The backend implements these as behaviors but the current UX treats them as a hierarchy (except for Admin which is a behavior in both). This is in part due to how we've chosen to do billing for bug reporting teams (only paying for debugger seats). I've talked with Jon about design options that better reflect the options available so I expect this will UI change in the future but for now it is trying to bridge that gap.

Reply:
I see. Ideally, I think we should show them in a consistent way (sounds like behaviors) rather than a mix, but as this is long-standing UX, I wouldn't block this PR on it.